public/app/ui/components/collections/library/detail.html

```diff
@@ -1,19 +1,7 @@
-<article id="library-detail" class="detail">
-    <!-- ko if: loading -->
-    <div class="loading text-muted">
-        <img src="/images/loading.gif" alt="" title="Loading animation" />
-        Loading Library record, please wait...
-    </div>
-    <!-- /ko -->
-    <!-- ko if: displayRecord -->
-    <div class="type-header">Library record</div>
-    <h1>
-        <span data-bind="component: displayFor('Title')"></span>
-        <span class="idno" data-bind="text: idno"></span>
-    </h1>
+<section class="detail detail-library">
```

Comment:
These templates have been split.  There is `app/ui/pages/detail/index.html` which is generic, and just has the header and then includes the relevant per-collection one of these, that are now at `app/ui/components/collections/[collection]/detail.html`.